azure-pipelines-official.yml

```diff
@@ -30,6 +50,23 @@ variables:
   - group: DotNet-DevDiv-Insertion-Workflow-Variables
   - name: _DevDivDropAccessToken
     value: $(dn-bot-devdiv-drop-rw-code-rw)
+  - group: ManagedLanguageSecrets
+
+  - name: BuildConfiguration
+    value: release
+  - name: Roslyn.GitHubEmail
+    value: dotnet-build-bot@microsoft.com
+  - name: Roslyn.GitHubToken
+    value: $(AccessToken-dotnet-build-bot-public-repo)
+  - name: Roslyn.GitHubUserName
+    value: dotnet-build-bot
+
+  - name: Insertion.InsertToolset
+    value: true
+  - name: Insertion.CreateDraftPR
+    value: true
+  - name: Insertion.TitlePrefix
+    value: '[Auto Insertion]'
```

Comment:
If we are creating drafts by default should we tag them as '[RPS Test] '. also do you need a trailing space?

Reply:
I felt that because the most common "scenario" in PublishData.json is a release branch, it's reasonable to default to making them a draft *and* tagging them `[Auto Insertion]`. I'm happy to adjust that if that doesn't feel right to you.

No need for a trailing space.

Reply:
Nah, go with it.